src/app/layout/navbar/navbar.component.html

```diff
@@ -1,5 +1,12 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <div class="container">
-    <a class="navbar-brand" href="#">GP4Tech</a>
+<nav class="navbar-fixed">
+  <div class="nav-wrapper">
+    <div class="container">
+      <a href="#" class="brand-logo left">GP4Tech</a>
+      <ul id="nav-mobile" class="left hide-on-med-and-down links">
+        <li>
+          <a>About Us</a>
+        </li>
+      </ul>
+    </div>
   </div>
-</nav>
\ No newline at end of file
+</nav>
```

Comment:
We could start adding the mobile sidenav here:
```
<ul class="sidenav" id="mobile-sidenav">
  <li><a class="sidenav-close">About Us</a></li>
</ul>
```
The sidenav needs to be initialize in order to work, but we could initialize it in the OnInit method of this component using auto init to avoid direct manipulation of the dom:
```
import * as Materialize from 'materialize-css';

Materialize.AutoInit();
```

Reply:
@mauricioarce yeap, I already have changed this in a separate branch, wanted to send it as a separate PR to include the full functionality for the sidenav. Not sure if you prefer to include it here?

Reply:
If it's in another branch I think it's fine 👍